### ISSUE TITLE
Getting a CI working

### ISSUE THREAD

**Issue by ddompe** [libreswitch/libreswitch#6]
Currently evaluating Travis and Shippable

**Comment by ddompe** [comment]
Travis is working now and deploying shared states into yocto.libreswitch.org... Seems to be working fine. Need to sync with @mzayats in final decision for this

**Comment by mzayats** [comment]
Shippable is not going to work, since it runs in Docker containers. Our test framework creates containers for the topology and doing docker in docker is a pain... So I guess Travis it is. I am going to contact Travis regarding caching.

**Comment by vitalivanov** [comment]
I think this can be closed now.

**Comment by ddompe** [comment]
No, we still need to get the testing regressions running... however that requires the single repo first...

**Comment by vitalivanov** [comment]
Additional note: travis-ci team increased build timeout for libreswitch project to 2 hours.
